docs/tool/parser.js

```diff
@@ -25,6 +25,29 @@ function isDocumentedNamedExport( node ) {
 	);
 }
 
+/**
+ * Returns true if the given exported declaration name is considered stable for
+ * documentation, or false otherwise.
+ *
+ * @see https://github.com/WordPress/gutenberg/blob/master/docs/contributors/coding-guidelines.md#experimental-and-unstable-apis
+ *
+ * @param {string} name Name to test.
+ *
+ * @return {boolean} Whether the provided name describes a stable API.
+ */
+const isStableExportName = ( name ) => ! /^__(unstable|experimental)/.test( name );
```

Comment:
Can we make it case insensitive? I'd like it to cover component names as well:

`__UnstableComponent` or `__ExperimentalComponent`.

Reply:
> Can we make it case insensitive? I'd like it to cover component names as well:
> 
> `__UnstableComponent` or `__ExperimentalComponent`.

AFAIK (and what I'd recommend), it's still capitalized as `__experimentalComponent`.

Example:

https://github.com/WordPress/gutenberg/blob/da09d36f20a26e3ba4c89ad8c1186a11284741b7/packages/data/src/index.js#L16

Reply:
Okay, let's follow this pattern 👍